lib/modules/dosomething/dosomething_signup/dosomething_signup.install

```diff
@@ -647,3 +673,39 @@ function dosomething_signup_update_7023() {
       ->execute();
   }
 }
+
+
+
+/**
+ * Adds fields to support competitions to the dosomething_signup_data_form table.
+ */
+function dosomething_signup_update_7024() {
+  $tbl_name = 'dosomething_signup_data_form';
+  // Load schema to get table definition.
+  $schema = dosomething_signup_schema();
+  // New fields to add.
+  $field_names = ['submit_text', 'skip_text', 'competition_signup'];
+  foreach ($field_names as $name) {
+    // If the field doesn't exist already:
+    if (!db_field_exists($tbl_name, $name)) {
+      // Add it per the schema field definition.
+      db_add_field($tbl_name, $name, $schema[$tbl_name]['fields'][$name]);
+    }
+  }
+}
+
+/**
+ * Adds competition column to the dosomething_signup table.
+ */
+function dosomething_signup_update_7025() {
+  $tbl_name = 'dosomething_signup';
+  // Load schema to get table definition.
+  $schema = dosomething_signup_schema();
+  // New field to add.
+  $field_name = 'competition';
+  // If the field doesn't exist already:
+  if (!db_field_exists($tbl_name, $field_name)) {
+    // Add it per the schema field definition.
+    db_add_field($tbl_name, $field_name, $schema[$tbl_name]['fields'][$field_name]);
+  }
+}
```

Comment:
Is there a reason this needs to be two separate updates instead of just adding `competitions` to the array of the update above?